nixos/modules/services/misc/bees.nix

```diff
@@ -0,0 +1,93 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.beesd;
+
+  fsOptions = with types; {
+    options.spec = mkOption {
+      type = str;
+      description = ''
+        Specification usable by findmnt; ie. LABEL=foo, UUID=bar, etc.
```

Comment:
Can you add maybe some more description and an example to this option? It's kind of unclear to me what's it for. (Example by setting `example = "<example value>";`)